File: fix_encoding.js
const fs = require('fs');
let content = fs.readFileSync('frontend/lib/screens/landing_screen.dart', 'utf-8');
let lines = content.split('\n');
for (let i = 0; i < lines.length; i++) {
  if (lines[i].includes('VETO AI')) {
    lines[i] = "            code == 'he' ? 'שאל את VETO AI' : code == 'ru' ? 'Спросить VETO AI' : 'Ask VETO AI',";
  }
}
fs.writeFileSync('frontend/lib/screens/landing_screen.dart', lines.join('\n'), 'utf-8');

File: fix_encoding.py
import sys

with open('frontend/lib/screens/landing_screen.dart', 'r', encoding='utf-8', errors='replace') as f:
    lines = f.readlines()

for i, line in enumerate(lines):
    if 'VETO AI' in line:
        lines[i] = "            code == 'he' ? 'שאל את VETO AI' : code == 'ru' ? 'Спросить VETO AI' : 'Ask VETO AI',\n"

with open('frontend/lib/screens/landing_screen.dart', 'w', encoding='utf-8') as f:
    f.writelines(lines)
